src/Commands/Self/Plugin/CreateCommand.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Self\Plugin;
+
+use Consolidation\AnnotatedCommand\CommandData;
+use Pantheon\Terminus\Exceptions\TerminusNotFoundException;
+use Pantheon\Terminus\Exceptions\TerminusException;
+use Pantheon\Terminus\Plugins\PluginInfo;
+
+/**
+ * Creates a new Terminus plugin using Composer.
+ * @package Pantheon\Terminus\Commands\Self\Plugin
+ */
+class CreateCommand extends PluginBaseCommand
+{
+    const USAGE_MESSAGE = 'terminus self:plugin:<create|new> <path';
```

Comment:
It's fine to have an alias `self:plugin:new` for `self:plugin:create`, but let's use only the later name in the usage text.

Reply:
Thanks. Fixed.